### ISSUE TITLE
Non-US keyboard layouts

### ISSUE THREAD

**Issue by Gaarco** [cloudhead/rx#89]
rx now works only with US layout and it was hard to figure out (I didn't see anything related on the website), may be possible to have support for different keyboard layouts?

**Comment by cloudhead** [comment]
See https://github.com/cloudhead/rx/pull/88

**Comment by Fnatte** [comment]
I use a Swedish layout and had problems with v0.4.0. The master-branch works for me, so I guess #88 or some other commit solved this issue.

**Comment by Fnatte** [comment]
I found that there's a difference between `Input::key` and `Input::Character` bindings. While #88 did change `:mode command` to a character binding, it did not change the other bindings. I solved this my in init.rx, like:

```
map         '/'             :zoom 1.0                                 --  Reset active view zoom
map/normal  '='             :brush/toggle perfect                     --  Pixel-perfect brush
map/normal  ']'             :brush/size +                             --  Increase brush size
map/normal  '['             :brush/size -                             --  Decrease brush size
map/visual  '\'             :selection/expand
```

That is, simply wrap problematic bindings with `'`.

**Comment by cloudhead** [comment]
Yeah, that's the way to go for now, since character bindings are not as powerful as key bindings (they don't support **release** events), I can't switch them all over.
